src/jackdaw/test/transports/rest_proxy.clj

```diff
@@ -210,6 +210,7 @@
   [config topic-metadata deserializers]
   (let [continue?   (atom true)
         xform       (comp
+                     #(assoc % :headers {}) ; cannot read headers over the Rest API
```

Comment:
do we need to pass it empty if it's not supported yet or can we just not set it?

Reply:
It can be just not set - I will add some docs though!